src/store/modules/interactiveMap.ts

```diff
@@ -33,6 +33,9 @@ export interface Card {
   organism: OrganismItem;
   modelId: number;
   method: string;
+  // `modified` is true when the user has made any change to the model, and it
+  // can be saved. Not relevant for data-driven cards.
```

Comment:
Not sure if user should be able to save the card when for example he changed the organism, didn't select the model and then closed the dialog, now we are getting 422

Reply:
Thanks, that shouldn't be possible. I'll add a fix for this